Modules/File/classes/class.ilObjFileGUI.php

```diff
@@ -319,7 +319,12 @@ public function executeCommand(): void
                     $this->addHeaderAction();
                     $ilTabs->clearTargets();
 
-                    parent::executeCommand();
+                    if (empty($cmd) || $cmd === 'render') {
+                        $cmd = Capabilities::INFO_PAGE->value;
+                        $this->$cmd();
```

Comment:
If we don't do this, the parent class will automatically call a `render` function, which is not given in `ilObjFileGUI` and lead to a crash.